code/modules/events/immovable_rod.dm

```diff
@@ -137,31 +137,37 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 		qdel(src)
 		qdel(other)
 
-/obj/effect/immovablerod/proc/penetrate(mob/living/L)
-	L.visible_message("<span class='danger'>[L] is penetrated by an immovable rod!</span>" , "<span class='userdanger'>The rod penetrates you!</span>" , "<span class='danger'>You hear a CLANG!</span>")
-	if(ishuman(L))
-		var/mob/living/carbon/human/H = L
-		H.adjustBruteLoss(160)
-	if(L && (L.density || prob(10)))
-		L.ex_act(EXPLODE_HEAVY)
-
-/obj/effect/immovablerod/attack_hand(mob/living/user)
```

Comment:
why're you removing the normal way to suplex an immovable rod?

Reply:
I dont see any point to keeping it if you just have to be on throw intent

Reply:
because people are going to try to suplex an immovable rod the normal way (which virtually requires you to NOT be on throw intent (because otherwise, throw intent will eat your click to turn itself off), as fulp is near-identical to /tg/ in most other areas, and why would someone have a reason to believe that fulp's changed how you suplex an immovable rod, of all things? fulp doesn't display its own changelog, remember?